src/dataframe.js

```diff
@@ -728,50 +728,21 @@ class DataFrame {
         return this.__newInstance__(this[__rows__], newColumnNames);
     }
 
-    /**
-     * Rename each column.
-     * @param {Array} newColumnNames The new column names of the DataFrame.
-     * @returns {DataFrame} A new DataFrame with the new column names.
-     * @example
-     * df.renameAll(['column1', 'column3', 'column4'])
-     */
-    renameAll(newColumnNames) {
```

Comment:
The renameAll must not be deleted but deprecated.
It means that the method must remain and will be deleted on the next major version (2.0.0).
Moreover the method must throw a warning when used.